js/src/wasm/WasmBaselineCompile.cpp

```diff
@@ -2264,10 +2264,12 @@ class BaseStackFrame final : public BaseStackFrameAllocator {
   }
 
   void storeImmediateF32ToStack(float imm, uint32_t destHeight, Register temp) {
-    union {
+    union StoredFloat32 {
       int32_t i32;
       float f32;
-    } bits = {.f32 = imm};
+
+      StoredFloat32(float f32) : f32(f32) {}
+    } bits{imm};
```

Comment:
remove blank.

Anonymous unions have certain special properties that could be erased by naming it. I think we can avoid doing so. A brace init of an anonymous union initializes its first member. So we can just swap the fields here:

```suggestion
    union {
      float f32;
      int32_t i32;
    } bits = {imm};
```

Reply:
Very interesting! Thanks for catching this one and for the helpful workaround.